apps/hash-graph/bin/cli/src/subcommand/snapshot.rs

```diff
@@ -49,36 +48,66 @@ pub async fn snapshot(args: SnapshotArgs) -> Result<(), GraphError> {
             report
         })?;
 
-    let mut store = pool
-        .acquire()
-        .await
-        .change_context(GraphError)
-        .map_err(|report| {
-            tracing::error!(error = ?report, "Failed to acquire database connection");
-            report
-        })?;
+    let mut store = SnapshotStore::new(
+        pool.acquire_owned()
```

Comment:
For simplicity, I changed this to `acquire_owned`. This will be reverted as soon as I removed channels again in the next PR.

Reply:
This was reverted in #2357